stock_scanner/scripts/export_scenario.py

```diff
@@ -90,7 +90,7 @@ def normalize_name(name):
 user_obj = Object(cnx, 'res.users')
 group_obj = Object(cnx, 'res.groups')
 scen_read = scenario_obj.read(
-    int(opts.scenario_id), [], {'active_test': False})
+    int(opts.scenario_id), [], '_classic_read', {'active_test': False})[0]
```

Comment:
@sylvain-garancher If teh id is not found  `[0]` raise `IndexError: list index out of range` and `if not scen_read:`will never be called
```python
scen_read = scenario_obj.read(
    int(opts.scenario_id), [], '_classic_read', {'active_test': False})
scen_read = scen_read and scen_read[0]
if not scen_read:
    ....
```